.flake8

```diff
@@ -1,15 +1,17 @@
 [flake8]
-# Exclude the grpc generated code
 exclude = ./.*,manim/grpc/gen/*
-max-complexity = 35
-max-line-length = 211
+max-complexity = 15
+max-line-length = 88
 statistics = True
+rst-roles = attr,class,func,meth,mod,obj,ref,doc,exc
+rst-directives = manim, SEEALSO, seealso
+select = B9
```

Comment:
Could you add some explanation via a comment as to what the purpose of these lines are?

Reply:
Sure